index.html

```diff
@@ -4,9 +4,21 @@
         <meta charset="utf-8">
         <title>Coinage</title>
     </head>
-    <body ng-controller="mainController">
-        
+    <body ng-controller="mainController as main">
+        <form name="form">
+            <input type="text" required name="amount" ng-model="amount" ng-pattern="pattern" />
+            <input type="button" value="Convert" ng-disabled="form.amount.$invalid"
+            ng-click="convertClicked(amount)">            
+        </form>   
+        <table style="width:100%">
```

Comment:
the functionality is in place why not show it? it is not much work and even if it doesn't have style it is more presentable that saying "the logic is in there but you can only see the result in a console log"